src/Parser/Hunter/Marksmanship/Modules/Features/VulnerableUptime.js

```diff
@@ -14,25 +14,35 @@ class VulnerableUpTime extends Analyzer {
     enemies: Enemies,
   };
 
+  get uptimeThreshold() {
+    return {
+      actual: this.uptimePercentage,
+      isLessThan: {
+        minor: 0.8,
+        average: 0.75,
+        major: 0.7,
+      },
+      style: 'percentage',
+    };
+  }
+
+  get uptimePercentage() {
+    return this.enemies.getBuffUptime(SPELLS.VULNERABLE.id) / this.owner.fightDuration;
+  }
   suggestions(when) {
-    const vulnerableuptime = this.enemies.getBuffUptime(SPELLS.VULNERABLE.id) / this.owner.fightDuration;
-
-    when(vulnerableuptime).isLessThan(0.80)
-      .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span>Your <SpellLink id={SPELLS.VULNERABLE.id} /> uptime can be improved. Make sure you use Windburst or Marked Shot to open the Vulnerable Window to maximize damage with your Aimed Shots</span>)
-          .icon(SPELLS.VULNERABLE.icon)
-          .actual(`${formatPercentage(actual)}% Vulnerable uptime`)
-          .recommended(`>${formatPercentage(recommended)}% is recommended`)
-          .regular(recommended - 0.05).major(recommended - 0.10);
-      });
+    when(this.uptimeThreshold).addSuggestion((suggest, actual, recommended) => {
+      return suggest(<span>Your <SpellLink id={SPELLS.VULNERABLE.id} /> uptime can be improved. Make sure you use Windburst or Marked Shot to open the Vulnerable Window to maximize damage with your Aimed Shots</span>)
```

Comment:
Since you are going back to rework these suggestions, I would update any `<span>` tags with `<Wrapper>` just to be consistent with the new style.